### PR TITLE
logging-{winston,bunyan}: accept serviceContext

### PR DESCRIPTION
**Pull request by ofrobots**
Accept serviceContext option at instantiation time. This is added to
error logs to allow Stackdriver to pick-up errors and associate with
the provided service.

For full details see: https://cloud.google.com/error-reporting/docs/formatting-error-messages

Fixes: https://github.com/GoogleCloudPlatform/google-cloud-node/issues/2513

### CODE REVIEW

File: packages/logging-bunyan/src/index.js [hunk 1]
```diff
@@ -56,6 +56,12 @@ var BUNYAN_TO_STACKDRIVER = {
  *     automatically, but you may optionally specify a specific monitored
  *     resource. For more information, see the
  *     [official documentation]{@link https://cloud.google.com/logging/docs/api/reference/rest/v2/MonitoredResource}
+ * @param {object=} options.serviceContext - For logged errors, we provide this
+ *     as the service context. For more information see
+ *     [this guide]{@link https://cloud.google.com/error-reporting/docs/formatting-error-messages}
+ *     and the [official documentation]{@link https://cloud.google.com/error-reporting/reference/rest/v1beta1/ServiceContext}.
+ * @param {string} options.serviceContext.service
```
Comment (stephenplusplus) [inline packages/logging-bunyan/src/index.js]: I think we should have some type of description here.

Comment (ofrobots) [reply packages/logging-bunyan/src/index.js]: Added.

**Comment by stephenplusplus** [comment]
Some linting issues: https://ci.appveyor.com/project/GoogleCloudPlatform/google-cloud-node/build/1.0.2421/job/4wve9049w8ar4as1#L40

**Comment by ofrobots** [comment]
Apologies. Fixed.

**Comment by stephenplusplus** [comment]
LG, thanks!

**Pull request merged**